### ISSUE TITLE
NextJS Support

### ISSUE THREAD

**Issue by joshcawthorne** [pacocoursey/cmdk#28]
Does this library officially support NextJS, or have any plans to? When trying to integrate it to a larger existing NextJS codebase, I received the following error: 

````
Unhandled Runtime Error
TypeError: react__WEBPACK_IMPORTED_MODULE_0__.useId is not a function

Call Stack
eval
node_modules/cmdk/dist/index.mjs (1:719)
````

However, when trying to make a minimal reproduction, I failed to get it to load at all, with Next complaining of a missing webpack loader -- not sure what loader is required in this instance.

See codesandbox here: [https://codesandbox.io/s/charming-darkness-gw9ezm](https://codesandbox.io/s/charming-darkness-gw9ezm)

TL;DR - Library doesn't seem to work with NextJS right now.

**Comment by pacocoursey** [comment]
Sounds like you're not using React 18, which is [required](https://github.com/pacocoursey/cmdk/blob/9db9a270515664e70f50b175ab8b8646e2bf01c5/cmdk/package.json#L17-L20) and [mentioned explicitly](https://github.com/pacocoursey/cmdk#faq).
